test/torch/tensors/test_native.py

```diff
@@ -191,6 +193,16 @@ def forward(self, x):
     ## Forward on the remote model
     pred = model_net(tensor_remote)
 
+    assert pred.is_wrapper
+    assert isinstance(pred.child, syft.PointerTensor)
+
+    model_net.get()
+
+    for p in model_net.parameters():
+        assert isinstance(p, torch.nn.Parameter)
+        assert not hasattr(p, "child")
+        break
```

Comment:
What? You only want one iteration?

Reply:
Yes I just need to check one parameters, they should all be equiavlent